cluster-scope/base/core/namespaces/chrisproject-dev/kustomization.yaml

```
@@ -0,0 +1,8 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: chrisproject-dev
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/limitranges/default
+    - ../../../../components/project-admin-rolebindings/chris-project
```

```suggestion
    - ../../../../components/project-admin-rolebindings/chris-project
    - ../../../../components/resourcequotas/small
```

include the quota please

---

Ack